config/checkstyle/checkstyle.xml

```diff
@@ -169,7 +169,7 @@
         <module name="IllegalInstantiation"/>
         <module name="InnerAssignment"/>
         <module name="MissingSwitchDefault"/>
-        <module name="RedundantThrows"/>
+        <!-- module name="RedundantThrows"   RedundantThrows was removed with version 6.2  -->
```

Comment:
Yes  we could.  I left in in and commented because this used to be supported, and someone might wonder why it is no longer included.